docs/eeps/draft-eep-purpose-and-guidelines.md

```diff
@@ -0,0 +1,204 @@
+
+Once the champion has asked the Embulk community as to whether an idea has any chance of acceptance, a draft EEP should be presented to the appropriate venue mentioned above. This gives the author a chance to flesh out the draft EEP to make properly formatted, of high quality, and to address initial concerns about the proposal.
+
+Submitting an EEP
+------------------
+
+Following the initial discussion above, the workflow varies based on whether any of the EEP's co-authors are in the core committers. If one or more of the EEP's co-authors are core committers, they are responsible for following the process outlined below. Otherwise (i.e. none of the co-authors are core committers), then the EEP author(s) will need to find a sponsor for the EEP.
+
+Ideally, a core committer sponsor is identified, but non-core commiter sponsors may also be selected from the core team based on a discussion in the core team. The sponsor's job is to provide guidance to the EEP author to help them through the logistics of the EEP process (somewhat acting like a mentor). Being a sponsor does **not** disqualify that person from becoming a co-author later on. The sponsor of a EEP is recorded in the "Sponsor:" field of the header.
+
+Once the sponsor or the core committer(s) co-authoring the EEP deem the EEP ready for submission, the proposal should be submitted as a draft EEP as a [pull request for Embulk's core Git repository](https://github.com/embulk/embulk/pulls). The draft must be written in the EEP style as described below, else it will fail review immediately (although minor errors may be corrected by the editors).
+
+The standard EEP workflow is:
+
+* You, the EEP author, fork [Embulk's core Git repository](https://github.com/embulk/embulk), and create a file named `draft-(summary-of-your-proposal).md` under `/docs/eeps/` that contains your new EEP.
+
+* In the "Type:" header field, enter "Standards Track", "Informational", or "Process" as appropriate, and for the "Status:" field enter "Draft". For full details, see the section of "EEP Header Preamble".
+
+* Push this to your GitHub fork, and submit it as a pull request.
+
+* The core team will review your pull request.
+  * If the reviewers conclude not to adopt the proposal at the moment, the pull request will be just declined.
+    * It normally should not happen because the pull request should already have passed the initial discussion above, and should have a sponsor.
+  * If the pull request does not seem ready for approval, the reviewers will send it back to the author for revision.
+  * Once the reviewers conclude to consider adopting the proposal, the pull request will be merged.
+    * The core committers may or may not assign your EEP a number when merging.
+    * A number will be assigned to the EEP, and its status will be "Accepted" immediately, if the core committers conclude that the EEP is promising enough. The filename needs to be renamed to `XXXX.md`, where "XXXX" is the assigned EEP number.
```

Comment:
I at first started with `eep-XXXX.md`, but I once removed `eep-` because the directory was `eeps`.

But, let me get it back to `eep-XXXX.md`. Thanks!